src/Composer/ComposerWidget.vala

```diff
@@ -787,4 +831,41 @@ public class Mail.ComposerWidget : Gtk.Grid {
             set_css_name (Gtk.STYLE_CLASS_ENTRY);
         }
     }
+
+    public override void destroy () {
+        if (discard_draft) {
+            base.destroy ();
+            return;
+        }
+
+        web_view.get_body_html.begin ((obj, res) => {
+            var body_html = web_view.get_body_html.end (res);
+
+            if (body_html == null || get_first_line_without_html (body_html) == first_line_of_message_without_html_on_load) {
```

Comment:
Can you explain what this is checking for? I don't understand what's going on with the whole `first_line_of_message_without_html` and what the significance is of that

Reply:
I tried to implement a way to detect whether the user changed anything or not. In case of a new draft this worked - but I removed the logic completely now, since thinking about it a bit more it will no longer work once we are able to edit drafts.